File: ProblemSet_14xx/Problem_1480/solution.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function(nums) {
    var result = [nums[0]];
    for (let i=1; i<nums.length; i++){
        result.push(nums[i] + result[result.length-1])
    }
    return result;
};

File: ProblemSet_14xx/Problem_1480/solution.cs
public class Solution
{
    public int[] RunningSum(int[] nums)
    {
        int n = nums.Length;
        int[] result = new int[n];
        
        if (n == 0) return result;

        result[0] = nums[0];
        for (int i = 1; i < n; i++)
        {
            result[i] = result[i - 1] + nums[i];
        }

        return result;
    }
}